src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -56,7 +66,7 @@ public static Logger getLogger(String name) {
      */
     public static <T> Logger getLogger(Class<T> clazz) {
         requireNonNull(clazz);
-        return getLogger(clazz.getSimpleName());
+        return getLogger(LOGGER_PREFIX + clazz.getSimpleName());
```

Comment:
What's the need to add a prefix?

Reply:
The way loggers are structured is they will all inherit from a parent logger (somewhat like a tree). 

Each logger can be represented by a path from root by seperating each name with a "." character.

So for instance, logger with name "parent.name1" is a logger whose parent is a logger with name "parent".

The root logger is a logger with name "".

While it is possible to change the root logger's log level to the current setting, it might lead to slow down of the application as the java package's log level might also change and we only want to set the log level of loggers from AB3. 

Therefore, a prefix is added to create a parent logger for AB3 where all loggers created from `LogsCenter` will inherit from it.

Reply:
I see. Let's add that explanation above this line so that future developers will know the reason.